### PR TITLE
Ab test cookie

### PR DESCRIPTION
**Pull request by danmassie**

### CODE REVIEW

File: app/com/gu/identity/frontend/filters/BetaUserGroupFilter.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class BetaUserGroupFilter @Inject() (configuration: Configuration)  extends Filter {
+
+  private[filters] val CookieName = "GU_PROFILE_BETA"
+  private[filters] val CookieValue = "1"
+
+  override def apply(nextFilter: (RequestHeader) => Future[Result])(request: RequestHeader): Future[Result] = {
+    val cookie = Cookie(
+      name = CookieName,
+      value = CookieValue,
+      domain = Some(configuration.identityCookieDomain),
```
Comment (jamespamplin) [inline app/com/gu/identity/frontend/filters/BetaUserGroupFilter.scala]: Can we use a different property to restrict the cookie to just `profile.theguardian.com`, such as `profileCookieDomain` or something. Also Cookie can be secure only too.

**Comment by jamespamplin** [comment]
Small note on Cookie domain, but otherwise looks good :+1:

File: app/com/gu/identity/frontend/filters/BetaUserGroupFilter.scala (hunk 1 above)
Comment (jamespamplin) [inline app/com/gu/identity/frontend/filters/BetaUserGroupFilter.scala]: Actually just remembered that we also have `profile-beta.theguardian.com` - so for now I think we can keep this on `.theguardian.com`.

Comment (danmassie) [inline app/com/gu/identity/frontend/filters/BetaUserGroupFilter.scala]: It won't work on the "beta" site if its profile.theguardian.com

**Comment by jamespamplin** [comment]
:+1: all good

**Pull request merged**